lib/chef/resource/chef_client_launchd.rb

```diff
@@ -110,21 +110,38 @@ class ChefClientLaunchd < Chef::Resource
           end
         end
 
-        launchd "com.chef.chef-client" do
+        launchd "com.#{Chef::Dist::SHORT}.#{Chef::Dist::CLIENT}" do
```

Comment:
yeah we need to get that chef-utils dist constants PR merged because you just broke it here

Reply:
Truth